### ISSUE TITLE
Pasting text bug

### ISSUE THREAD

**Issue by vleango** [hanspinckaers/GrowingTextView#51]
When pasting text, HPGrowingTextView grows, however, the textView is unscrollable, and sometimes the text is cut off.

I did find that after dismissing the keyboard and having HPGrowingTextView become the firstResponder again fixes this issue, but I hope there's a solution that doesn't require this.

**Comment by vleango** [comment]
a little update:

When selecting All, and deleting all text, the HPGrowingTextView is no longer scrollable (even with dismissing the keyboard and becomingFirstResponder). This makes sense, since it's not a multi-lined textview at present. However, when text is pasted back in, it is still not scrolling

**Comment by vleango** [comment]
took me a while to figure it out.

What I did was reset the scrolling in: 

growingTextView didChangeHeight:(float)height;

**Comment by vleango** [comment]
growingTextView didChangeHeight:(float)height; was working for the simulator, but I found:

growingTextViewDidChange

to be working for both the simulator and device

**Comment by kejinlu** [comment]
How to reset the scrolling？

**Comment by kejinlu** [comment]
I think this is a bug

**Comment by kejinlu** [comment]
@HansPinckaers

**Comment by kejinlu** [comment]
iOS7 With XCode5

**Comment by schemers** [comment]
I just ran into the same issue. reproducible in the growing text example project by pasting about 8-9 lines of text (enough lines to be higher than max height), like:

1\n
2\n
...
8\n

pasting that causes most of 7 and 8 to be clipped.

I came up with a fix by starting with scrollEnabled = YES in commentInitializer, as well as setting it to YES in 
- (BOOL)textView:shouldChangeTextInRange:replacementText:

additionally, I had to add same "scoll to caret" logic in growDidStop, otherwise the caret would sometimes be wrong.

still doing some testing to make sure everything else is working, but that seems to fix it (at least inside my app, I haven't tested inside of sample app yet).

need to set scrollEnabled = YES only on iOS7, causes bad things on iOS6.
